packages/react-charting/src/components/GaugeChart/GaugeChart.test.tsx

```diff
@@ -0,0 +1,305 @@
+import * as React from 'react';
```

Comment:
Could you add the test strategy doc as well.

Reply:
done